.github/workflows/continuous-integration.yml

```diff
@@ -1,10 +1,6 @@
 name: Check build pass/fail
 
-on:
-  pull_request:
-    branches:
-      - master
-  push:
+on: [pull_request, push]
```

Comment:
Won't this make the CI build twice? Because a pull request is a push. It's been building twice in this branch

Reply:
Will fix